src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,102 @@
+import java.util.Collections;
+import java.util.List;
+
+public class Lotto {
+    private static int THREE_CORRECT_PRIZE = 5000;
+    private static int FOUR_CORRECT_PRIZE = 50000;
+    private static int FIVE_CORRECT_PRIZE = 1500000;
+    private static int FIVE_CORRECT_PRIZE_WITH_BONUS = 30000000;
+    private static int ALL_CORRECT_PRIZE = 2000000000;
+    private static int THREE_CORRECT_NUMBER_INDEX = 0;
+    private static int FOUR_CORRECT_NUMBER_INDEX = 1;
+    private static int FIVE_CORRECT_NUMBER_INDEX = 2;
+    private static int FIVE_CORRECT_WITH_BONUS_NUMBER_INDEX = 3;
+    private static int ALL_CORRECT_NUMBER_INDEX = 4;
+    private int money;
+
+    private LottoLists<ArrayList<Integer>> lottoLists;
+
+    public void lottoGame(){
+        this.money = Input.inputMoney();
+        int numberOfLotto = getNumberOfLotto(money);
+        Output.outputNumberOfPurchaseLotto(numberOfLotto);
+
+        lottoLists = new LottoLists(getPurchasesLotto(numberOfLotto));
+        Output.outputLottoLists(lottoLists);
+
+        Integer[] previousLottoNumber = Input.inputPreviousLottoNumber();
+        int bonusBall = Input.inputBonusBall();
+        Integer[] lottoResult = compareLottoNumber(lottoLists,previousLottoNumber,bonusBall);
+        double yield = getYield(lottoResult);
+        Output.outputStatistics(lottoResult,yield);
+    }
+
+    private int getNumberOfLotto(int money) {
+        return money/1000;
+    }
+    private List<ArrayList<Integer>> getPurchasesLotto(int numberOfLotto){
+        List<ArrayList<Integer>> createLottoLists = new ArrayList<>();
+        for(int i =0;i<numberOfLotto;i++) createLottoLists.add(getLottoNumber());
+        return createLottoLists;
+    }
+
+    private ArrayList<Integer> getLottoNumber(){
+        List<LottoNumber> allBalls = new ArrayList<>(Arrays.asList(LottoNumber.values()));
+        Collections.shuffle(allBalls);
+        ArrayList<Integer> lottoList = new ArrayList<>();
+        for(int i =0;i<6;i++){
+            lottoList.add(allBalls.get(i).getNumber());
+        }
+        return lottoList;
+    }
+
+    private Integer[] compareLottoNumber(LottoLists<ArrayList<Integer>> lottoLists,Integer[] previousLottoNumber,int bonusBall){
+        Integer[] statistics = new Integer[]{0,0,0,0,0};
+        for(int i=0;i<lottoLists.size();i++){
+            ArrayList<Integer> lottoList= lottoLists.getLottoList(i);
+            int numberCnt = 0;
+            boolean isCorrectBonus = false;
+
+            numberCnt = checkNumber(lottoList,previousLottoNumber,numberCnt);
+            isCorrectBonus = checkBonus(lottoList, bonusBall, isCorrectBonus);
+            setStatistics(statistics,numberCnt,isCorrectBonus);
+        }
+        return statistics;
+    }
+
+    private int checkNumber(ArrayList<Integer> lottoList,Integer[] previousLottoNumber,int numberCnt){
+        for (Integer previousNumber : previousLottoNumber) {
+            numberCnt = getNumberCnt(lottoList, numberCnt, previousNumber);
```

Comment:
`LottoList`라는 클래스가 생긴다면, 
```suggestion
            numberCnt = lottoList.getNumberCnt(numberCnt, previousNumber);
```
로 변화가 되고, 아래에 있는 `getNumberCnt` 메소드의 책임이 `LottoList`에게 가겠죠?
지금 `Lotto`클래스의 역할이 너무 많다고 생각하는데 어떠신가요?